modules/contact-form/admin.php

```diff
@@ -902,3 +907,73 @@ function grunion_recheck_queue() {
 }
 
 add_action( 'wp_ajax_grunion_recheck_queue', 'grunion_recheck_queue' );
+
+/**
+ * Delete a number of spam feedbacks via an AJAX request.
+ */
+function grunion_delete_spam_feedbacks() {
+	if ( ! wp_verify_nonce( $_POST['nonce'], 'jetpack_delete_spam_feedbacks' ) ) {
+		wp_send_json_error(
+			__( 'You don&#8217;t have permission to do that.', 'jetpack' ),
+			403
+		);
+
+		return;
+	}
+
+	$deleted_feedbacks = 0;
+
+	$delete_limit = 25;
+	$delete_limit = apply_filters( 'jetpack_delete_spam_feedbacks_limit', $delete_limit );
```

Comment:
Let's add a docblock here so the parser can create a Codex page from this:

```suggestion
	/**
	 * Filter the amount of Spam feedback one can delete at once.
	 *
	 * @module contact-form
	 *
	 * @since 8.6.0
	 *
	 * @param int $delete_limit Number of spam to process at once. Default to 25.
	 */
	$delete_limit = apply_filters( 'jetpack_delete_spam_feedbacks_limit', $delete_limit );
```